### PR TITLE
charts: add kommander

### PR DESCRIPTION
**Pull request by hectorj2f**

### CODE REVIEW

File: stable/kommander/values.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+kommander:
+  replicas: 1
+  image:
```
Comment (alejandroEsc) [inline stable/kommander/values.yaml]: pull policy would be nice to have here.

Comment (alejandroEsc) [inline stable/kommander/values.yaml]: I would suggest image would be a first level field

File: stable/kommander/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
+---
```
Comment (alejandroEsc) [inline stable/kommander/templates/deployment.yaml]: Can we move service account (roles and bindings) information to another file.

File: stable/kommander/templates/deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+apiVersion: v1
+kind: Namespace
```
Comment (alejandroEsc) [inline stable/kommander/templates/deployment.yaml]: Hmmm, i honestly forget if the namespace is created or automagically for you.

File: stable/kommander/values.yaml (hunk 1 above)
Comment (hectorj2f) [reply stable/kommander/values.yaml]: Moved

Comment (hectorj2f) [reply stable/kommander/values.yaml]: ack

File: stable/kommander/templates/deployment.yaml (hunk 2 above)
Comment (hectorj2f) [reply stable/kommander/templates/deployment.yaml]: Done

File: stable/kommander/templates/deployment.yaml (hunk 3 above)
Comment (hectorj2f) [reply stable/kommander/templates/deployment.yaml]: It should get all created at the same time.

**Comment by joejulian** [comment]
Rats... I was too slow... You don't normally create namespaces in charts, you use the `Release.Namespace` variable to reference the namespace in which this chart was installed.

**Pull request merged**